src/drawing/DrawingPath.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { v4 as uuidv4 } from 'uuid';
+import { white } from 'box-ui-elements/es/styles/variables';
+import { DrawingSVGRef } from './DrawingSVG';
+import { PathGroup } from '../@types';
+import { Point } from '../region/transformUtil';
+import { isVectorEffectSupported } from './drawingUtil';
+import './DrawingPath.scss';
+
+export type Props = {
+    isActive?: boolean;
+    pathGroup: PathGroup;
+    rootEl: DrawingSVGRef | null;
+};
+
+export const DRAWING_TARGET_BORDER = 1;
+
+export const getPathCommands = (points: Point[]): string => {
+    if (!points || !points.length) {
+        return '';
+    }
+
+    const { x: startX, y: startY } = points[0];
+    const d = points
+        .map(({ x, y }, index, array) => {
+            const prevPoint = array[index - 1];
+            const nextPoint = array[index + 1];
+            if (!prevPoint || !nextPoint) {
+                return '';
+            }
+
+            const xc1 = (x + prevPoint.x) / 2;
+            const yc1 = (y + prevPoint.y) / 2;
+            const xc2 = (x + nextPoint.x) / 2;
+            const yc2 = (y + nextPoint.y) / 2;
+
+            return `C ${xc1} ${yc1}, ${x} ${y}, ${xc2} ${yc2}`;
+        })
+        .join(' ');
+
+    return `M ${startX} ${startY} ${d}`;
+};
+
+export const DrawingPath = (props: Props): JSX.Element => {
+    const {
+        isActive = false,
+        pathGroup: {
+            paths,
+            stroke: { color, size },
+        },
+        rootEl,
+    } = props;
+
+    let strokeWidth = size;
+    let strokeBorderWidth = size + DRAWING_TARGET_BORDER * 2;
```

Comment:
The math here is a little unclear. What is the reason for the multiple? Should this be `size + 2`? Or `(size + 1) * 2`?

Reply:
A line has two sides and each side needs a border, so add BORDER_WIDTH * 2

Reply:
If the question arises during review, it will likely arise again in a few months when someone needs to modify the code. Can we add a comment to the code explaining the reasoning?